src/Templates/ProjectTemplates/CSharp/.NETCore/CSharpConsoleApplication/ProjectTemplate.csproj

```diff
@@ -1,5 +1,4 @@
-﻿<Project ToolsVersion="15.0">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+﻿<Project Sdk="Microsoft.Net.Sdk">
```

Comment:
We still need ToolsVersion. Not having it makes VS pop up an upgrade dialog at the beginning and the change to fix that is not yet in. Let's separate the removal of ToolsVersion into a separate change